console/frontend/src/main/frontend/src/app/views/logging/logging-add/logging-add.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="wrapper wrapper-content animated fadeInRight">
+  <div class="alert alert-warning" role="alert" type="warning" *ngIf="error">
+    <span [innerHTML]="error"></span>
```

Comment:
Because it can be null which shouldn't be converted to a string 'null' but instead not do anything at all
The ngIf covers this regardless but better safe than sorry when mixing data types

Reply:
Somehow, I totally overlooked that. Thank you.